feder/institutions/forms.py

```diff
@@ -1,13 +1,28 @@
-from atom.ext.crispy_forms.forms import SingleButtonMixin
 from braces.forms import UserKwargModelFormMixin
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Layout, Fieldset, ButtonHolder, Submit
 from dal import autocomplete
 from django import forms
+from django.utils.translation import ugettext_lazy as _
 
 from .models import Institution
 
 
-class InstitutionForm(SingleButtonMixin, UserKwargModelFormMixin, forms.ModelForm):
+class InstitutionForm(UserKwargModelFormMixin, forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Form "helper" is needed to have control over rendering by "crispy" tag.
+        self.helper = FormHelper()
+        self.helper.layout = Layout(
+            Fieldset("", "name", "jst", "regon", "email"),
+            Fieldset("", "tags", css_class="form-group scrollable-widget"),
+            ButtonHolder(Submit("submit", _("Update"), css_class="btn btn-primary")),
+        )
```

Comment:
I suggest follow notation from other part of project: https://github.com/watchdogpolska/feder/blob/a25820deea70d09187e6fc17e91046826d0de475/feder/monitorings/forms.py#L19-L22
It allows skip `FormHelper` implicit definition and use implicit submit button definition.

Reply:
Ok, change applied.